addons/website/static/src/less/website.ui.less

```diff
@@ -4,14 +4,16 @@
 
 // LAYOUTING
 body.o_connected_user {
-    padding-top: @odoo-navbar-height!important;
+    padding-top: @odoo-navbar-height - 2!important;
```

Comment:
navbar.less is ugly but the `- 2` there are for navbar internal elements. For an outside element as the `body`, it should not be `- 2`. If the border is removed, then it should also be removed in the backend and so all the `- 2` should be removed everywhere.